docs/_articles/en/v3.x/salesforce-graph-engine/rules.md

```diff
@@ -37,10 +37,43 @@ ApexFlsViolationRule detects [Create, Read, Update, and Delete (CRUD) and Field-
 |				 |SOQL queries that use [WITH SECURITY_ENFORCED](https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/apex_classes_with_security_enforced.htm#apex_classes_with_security_enforced)|
 |				 |Lists filtered by [Security.stripInaccessible](https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/apex_classes_with_security_stripInaccessible.htm)|
 
-Have a look at related [FAQ](./en/v3.x/faq/#questions-about-interpreting-apexflsviolationrule-results) to understand the results generated.
+### Interpreting ApexFlsViolationRule Results
+
+Match any violation message that you receive with these scenarios to understand more about the violation.
+
+*Common Scenario*
```

Comment:
I know this is copying the original text, but for some reason I'm not in love with "scenario". The thing they're matching is violation text, and "scenario" seems to imply a broader scope than that. Is there a different word we can use? "Case" maybe?

Reply:
@jfeingold35 So below I use scenario again - Additional Clause Scenario, stripInaccessible Scenario - so are you suggesting that for each we rename to Case?
Common Cases
Additional Clause Case
stripInaccessible Case

and so on?

Reply:
Yeah, I'd say we should be using "case" (or something similar) instead of "scenario" throughout this section. (But I'm prepared to acknowledge the possibility that I'm being fussy over nothing here)

Reply:
@rmohan20 Do you have a preference: Case or Scenario throughout?